ja_JP/contributing.md

```diff
@@ -0,0 +1,46 @@
+# AVAにコントリビュート
```

Comment:
As the [French version](https://github.com/sindresorhus/ava-docs/edit/master/fr_FR/contributing.md) and [Spanish version](https://github.com/sindresorhus/ava-docs/edit/master/es_ES/contributing.md), add a header for the link to the original and to have a link on the differences, for example markdown formatted (Note, I do not speak Japanese, it is a translation in english. It is therefore necessary to translate it into Japanese.) :

```markdown
___
**Translator's Note**

This is the translation of [contributing.md](https://github.com/sindresorhus/ava/blob/master/contributing.md) file. Here is a [link](https://github.com/sindresorhus/ava/compare/b76f3d513a41edd876f70fc3e61d7d968cd531a6...master#diff-cc4aac3e9be04e0413c9520f223b493c) to differences with the master AVA branch (If clicking the link, no changes were found in the `contributing.md` file, will be that the translation is up to date).
___
```

1. Where the link `contributing.md` points to the file contributing.md from AVA
2. Where the link `link` points to the last commit respective translation (`65ae07c76b2a5927e6bcc00630a691c98f6e7c46`) followed by `...master#diff-` and sha file (`cc4aac3e9be04e0413c9520f223b493c`)

That is to do on the 6 files : code-of-conduct, contributing.md, maintaining.md, readme.md, code-coverage and endpoint-testing.